examples/cxx/domains/src/publisher.cpp

```diff
@@ -68,9 +68,9 @@ auto main(int argc, char** argv) -> int {
 
         auto sample = publisher.loan_uninit().expect("acquire sample");
 
-        sample.write_payload(TransmissionData { counter, counter * 3, counter * 812.12 }); // NOLINT
+        auto initialized_sample =
```

Comment:
Can you please adjust the example in the iceoryx2 readthedocs c++ documentation here: https://github.com/eclipse-iceoryx/iceoryx2/blob/main/internal/cpp_doc_generator/cxx_pub_sub.rst